pkg/config/tang.go

```diff
@@ -8,6 +8,7 @@ import (
 var Tang *tangy.Tangy
 
 func ConfigureTang() error {
+	log.Info().Msgf("LOGHERE: pulp server: %v, snapshots: %v", Get().Clients.Pulp.Server, Get().Features.Snapshots.Enabled)
```

Comment:
did you want to keep these logs or are they remnants from debugging?

Reply:
oh whoops

Reply:
removed the logs!